torch/_inductor/config.py

```diff
@@ -25,6 +25,11 @@ def autotune_remote_cache_default() -> Optional[bool]:
     return None
 
 
+# Enable auto_functionalized_v2 (enabled by default)
+enable_auto_functionalized_v2 = (
+    os.environ.get("TORCHDYNAMO_AUTO_FUNCTIONALIZED_V2", "1") == "1"
+)
```

Comment:
It might be better to start at disabled by default and then enable it by default in a future PR. That way when someone does a bisection + revert, they will revert the config change instead of this PR.

Reply:
ok

Reply:
Or, if it's easier, a dummy PR on top of this that disables it and then another one that enables it lol

Reply:
I will start with disabled.